app/Repositories/SignupRepository.php

```diff
@@ -43,6 +45,10 @@ public function create($data)
             $signup->save(['timestamps' => false]);
```

Comment:
Sorry i know this isn't a part of you PR, but is this saving the signup without timestamps? Why do we want to do this?